beta/experience-platform-extension/tutorials/tutorial-2-swift-core-setup.md

```diff
@@ -0,0 +1,116 @@
+# Tutorial 2 - Swift Core setup
+
+This tutorial illustrates how you may send XDM commerce events to Adobe Experience Platform via Experience Edge using the AEP Edge extension in a sample application, provided to you in iOS \(Swift\) and Android.
+
+The demo mobile application has multiple tabs. For this exercise, the `Edge` and `Assurance` tabs will be used, demonstrating XDM commerce events in a mobile application.
+
+## Prerequisites for this tutorial
+
+- Access to Adobe Experience Launch dashboard
+- Xcode 12
+- Cocoapod
+
+### Configure the Launch Mobile property
+
+To finish the tutorial, you can either use a existing Mobile Property and create a new one. Follow the below steps if you need to create a new Mobile property in Launch UI:
+
+1. First, follow these steps to [Set up a mobile property](https://aep-sdks.gitbook.io/docs/getting-started/create-a-mobile-property) in Adobe Experience Platform Launch.
+
+2. Go to the Publishing Flow menu, select the development library you created and click `Add All Changed Resources`. 
+3. Click `Save & Build for Development` to publish the changes in the **Development** environment.
+
+### Download the sample app
+
+Clone or download the iOS Swift Sample application from https://github.com/adobe/aepsdk-sample-app-ios.
+
+To get started, follow the steps described in [AEP SDK Sample App Swift - Installation](https://github.com/adobe/aepsdk-sample-app-ios#installation).
+
+### Install the AEPCore SDK
+
+Add the following code to the `Podfile`
+
+```
+
+  pod 'AEPServices', :git => 'git@github.com:adobe/aepsdk-core-ios.git', :branch => 'main'
+  pod 'AEPCore', :git => 'git@github.com:adobe/aepsdk-core-ios.git', :branch => 'main'
+  pod 'AEPLifecycle', :git => 'git@github.com:adobe/aepsdk-core-ios.git', :branch => 'main'
+  pod 'AEPIdentity', :git => 'git@github.com:adobe/aepsdk-core-ios.git', :branch => 'main'
+  pod 'AEPSignal', :git => 'git@github.com:adobe/aepsdk-core-ios.git', :branch => 'main'
+  pod 'AEPRulesEngine', :git => 'git@github.com:adobe/aepsdk-rulesengine-ios.git', :branch => 'main'
+```
+
+Run `pod install`, now it should download all the dependencies and add them to the project.
+
+Open the project from `AEPSampleApp.xcworkspace`
+
+
+
+### Init SDK
+
+1. Open the AppDelegate.swift, add the following import statements.
+
+```
+import AEPCore
+import AEPLifecycle
+import AEPIdentity
+import AEPSignal
+```
+
+2. In the method ` func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool` , register `Lifecycle` , `Identity` and `Signal` extensions and start the SDK. Also call the `MobileCore.configureWith(appId:)` to configure the app id. 
+
+```
+
+        MobileCore.setLogLevel(level: .trace)
```

Comment:
1. let's make sure the format of this code is readable
2. probably good to explain why they're setting log level